### DISCUSSION TITLE
issue: Combobox has dropdown icon overlap

### DISCUSSION THREAD

**Discussion by nickfun** [twilio-labs/paste#890] [General] [answered]
`<Combobox>` when displaying the list of options any other down-arrow icons will display over the list.

Expand the "Forms Example" on here: https://codesandbox.io/s/paste-starter-kit-forked-5cpvl

shots:
<img width="696" alt="Screen Shot 2020-10-27 at 12 04 17 PM" src="https://user-images.githubusercontent.com/1335784/97349722-0d52a300-184d-11eb-9186-20e2223b4ba9.png">

overlap when open:
<img width="718" alt="Screen Shot 2020-10-27 at 12 04 25 PM" src="https://user-images.githubusercontent.com/1335784/97349733-0fb4fd00-184d-11eb-90e5-b5e64c6f301b.png">

our workaround: https://code.hq.twilio.com/react/console-ahoy-v2/pull/24

**Comment by SiTaggart** [comment]
Hi @nickfun, Thanks for the bug report. Definitely looks like a simple z-index issue with the combobox listbox. We'll add it to the backlog and get a fix out next sprint.

**Comment by serifluous** [comment, marked as answer]
Hey @nickfun ! We just merged in a fix for this (#917) Thanks, again.
